packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -86,25 +96,19 @@ export default class HostItem extends Component {
                 &nbsp;&nbsp;
                 <Tooltip
                   content={
-                    isEmpty(host.current_operations) ? (
-                      toolTipContent
-                    ) : (
-                      <div>
-                        {toolTipContent}
-                        {' ('}
-                        {map(host.current_operations)[0]}
-                        {')'}
-                      </div>
-                    )
+                    <span>
+                      {_(`powerState${state}`)}
+                      {state === 'Busy' && (
+                        <span>
```

Comment:
I suggest

```html
<span> ({Object.values(host.current_operations)[0]})</span>
```

Reply:
Agreed, but I'm trying to change as few things as possible (even though it doesn't look like it) and this change isn't related to the PR so I'll fix it in another one.